### ISSUE TITLE
[Feature Request 3] for main move path : add second move path, third move path, etc.. (PV)

### ISSUE THREAD

**Issue by wonderingabout** [Tencent/PhoenixGo#85]
Hi :+1: 

PhoenixGo native engine support is being added to goreviewpartner software here [#96](https://github.com/pnprog/goreviewpartner/pull/96)

while we are working on this, we noticed that the `main move path` behaviour can be improved, by adding `second move path`, `third move path`
because goreviewpartner is an automated review tool that will send main move path, second move path, third move path, etc... into a .rsgf file to view it later after analysis

the new idea is to add more data to verbose levels, something like this : 

```
I0121 17:55:30.734530 30129 mcts_debugger.cc:43] ========== debug info for 54th move(w) begin ==========
I0121 17:55:30.734563 30129 mcts_debugger.cc:44] main move path: kd(7163,-0.77,0.14,-0.85),be(1775,0.78,0.13,0.80),bd(1171,-0.76,0.41,-0.84),de(476,0.75,0.48,0.76),cd(472,-0.75,0.91,-0.81),if(90,0.76,0.16,0.80),kf(50,-0.76,0.50,-0.84),eq(14,0.76,0.22,0.75),jc(4,-0.72,0.19,-0.73),gf(1,0.70,0.25,0.70),op(0,-nan,0.00,nan)
I0121 17:55:30.734563 30129 mcts_debugger.cc:44] second move path: ch(1374,-0.77,0.14,-0.85),kd(1775,0.78,0.13,0.80),bd(1171,-0.76,0.41,-0.84),de(476,0.75,0.48,0.76),cd(472,-0.75,0.91,-0.81),if(90,0.76,0.16,0.80),kf(50,-0.76,0.50,-0.84),eq(14,0.76,0.22,0.75),jc(4,-0.72,0.19,-0.73),gf(1,0.70,0.25,0.70),op(0,-nan,0.00,nan)
I0121 17:55:30.734563 30129 mcts_debugger.cc:44] third move path: di(394,-0.77,0.14,-0.85),de(1775,0.78,0.13,0.80),bd(1171,-0.76,0.41,-0.84),kd(476,0.75,0.48,0.76),kf(472,-0.75,0.91,-0.81),if(90,0.76,0.16,0.80),kf(50,-0.76,0.50,-0.84),eq(14,0.76,0.22,0.75),jc(4,-0.72,0.19,-0.73),gf(1,0.70,0.25,0.70),op(0,-nan,0.00,nan) 
I0121 17:55:30.734592 30129 mcts_debugger.cc:139] kd: N=7163, W=-5490.75, Q=-0.766544, p=0.14197, v=-0.850747
I0121 17:55:30.734598 30129 mcts_debugger.cc:139] ch: N=1374, W=-1131.4, Q=-0.823437, p=0.338758, v=-0.843087
I0121 17:55:30.734604 30129 mcts_debugger.cc:139] di: N=394, W=-325.479, Q=-0.826088, p=0.10128, v=-0.769427
I0121 17:55:30.734617 30129 mcts_debugger.cc:139] bg: N=293, W=-243.426, Q=-0.830806, p=0.0814862, v=-0.790699
I0121 17:55:30.734627 30129 mcts_debugger.cc:139] bp: N=261, W=-220.618, Q=-0.84528, p=0.0871321, v=-0.829451
I0121 17:55:30.734632 30129 mcts_debugger.cc:139] bh: N=219, W=-191.466, Q=-0.874273, p=0.0987152, v=-0.857132
I0121 17:55:30.734642 30129 mcts_debugger.cc:139] bi: N=73, W=-63.9489, Q=-0.876012, p=0.033503, v=-0.802584
I0121 17:55:30.734650 30129 mcts_debugger.cc:139] eh: N=66, W=-52.9326, Q=-0.802009, p=0.0106665, v=-0.865476
I0121 17:55:30.734656 30129 mcts_debugger.cc:139] cp: N=64, W=-52.9554, Q=-0.827427, p=0.0165746, v=-0.828988
I0121 17:55:30.734663 30129 mcts_debugger.cc:139] cq: N=21, W=-18.6035, Q=-0.885883, p=0.0106966, v=-0.872695
I0121 17:55:31.220618 30129 mcts_debugger.cc:48] model global step: 639200
I0121 17:55:31.220625 30129 mcts_debugger.cc:49] ========== debug info for 54th move(w) end   ==========
```

we can consider showing it for forth move path, fifth move path, etc... up to tenth move path, but i am not sure that this is very accurate (and probably not very needed) because last moves usually have low simulations

to sum up, i think it would be great to add `second move path` and `third move path` for additional analysis information

should we add it to the existing `v=1` ? or should we create a new verbose level to do that ?

@wodesuck @pnprog

**Comment by wodesuck** [comment]
support in [this commit](https://github.com/Tencent/PhoenixGo/commit/e640de5fb3253a8bd44ffb482b07f6e53ba8cc4f).

**Comment by wonderingabout** [comment]
@pnprog
for GoreviewPartner, see this

@wodesuck 
big thanks again :100:

**Comment by wonderingabout** [comment]
i think we can close this issue now, thanks again, really :1st_place_medal:
